Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.js

```diff
@@ -9,42 +9,96 @@ class SimCachePanel extends UIElement {
     connectedCallback() {
         this.m_connected = true;
 
-        this.m_ingameUI = this.querySelector( "ingame-ui" );
-        this.m_ingameUI.addEventListener( "panelActive", () => this.onPanelActive() );
+        this.m_ingameUI = this.querySelector("ingame-ui");
```

Comment:
Didn't like the ( spacing )? 😛

Reply:
Same as the previous comment -- VSCode removed it upon formatting.  I do like `( spacing )` for readability now, but idk how to get the formatter set up to preserve it. :P